skrub/_agg_joiner.py

```diff
@@ -82,7 +82,7 @@ class AggJoiner(BaseEstimator, TransformerMixin):
     The grouping columns used during the aggregation are the columns used
     as keys for joining.
 
-    Accepts :obj:`pandas.DataFrame` and :obj:`polars.DataFrame` inputs.
+    Accepts :obj:`pandas.DataFrame` and :class:`polars.DataFrame` inputs.
```

Comment:
I guess the same change could be done in aggtarget

btw if we want to specify in all skrub docstrings if polars inputs are accepted, that info is missing from many of them